oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -207,6 +207,16 @@ Frontend Technology Selection
 
 #. **CSS should be generated using SCSS**
 
+   .. note::
+
+      While there are no current plans to deprecated SCSS, the recommendation
+      as of February 2025 is to use CSS variables instead of SCSS variables
+      wherever possible now that browsers support them directly.
+
+      Given that browsers support CSS nesting natively too, there is not so much
+      need for SCSS in general either, but we probably still need it since bootstrap
+      and thus Paragon are built using it.
```

Comment:
I'm not sure we need this part. It feels to me like it takes focus away from the variables.

Reply:
I agree. It is also slightly implying that Paragon doesn't use CSS variables, whereas `v23` is largely the primary driver for migrating to CSS variables across the MFEs (despite still using SCSS for things other than variables).

Reply:
I would also probably drop this paragraph, since support for CSS nesting is not yet at a high enough level. According to [Can I use](https://caniuse.com/css-nesting), CSS nesting support is 91%, meaning 9% of users will not be able to fully use web applications.